src/Microsoft.Azure.CosmosRepository/InMemoryRepository.cs

```diff
@@ -29,20 +28,17 @@ internal class InMemoryRepository<TItem> : IRepository<TItem> where TItem : clas
     {
         internal long CurrentTs => DateTimeOffset.UtcNow.ToUnixTimeSeconds();
         internal ConcurrentDictionary<string, string> Items { get; } = new();
-        internal ISpecificationEvaluator _specificationEvaluator { get; set; }
+
+        private readonly ISpecificationEvaluator _specificationEvaluator;
```

Comment:
For private fields, let's try to keep those at the top of the file, then internals.

Reply:
Will ammend.